### PR TITLE
new module bref3

### PR DESCRIPTION
**Pull request by xfzhao**

### CODE REVIEW

File: var/spack/repos/builtin/packages/bref3/package.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+from spack import *
+import os.path
+
+
+class Bref3(Package):
+    """Bref3: Converts from VCF format to bref3 format."""
+
+    homepage = "http://faculty.washington.edu/browning/beagle/beagle.html"
+
+    version('25Nov19', sha256='969c0881050c4a48d19be9ea64bf49fa68c1403b69f9f739bbfd865dda639b2d',
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/bref3/package.py]: These versions won't sort correctly, I would use:
```suggestion
    version('2019-11-25', sha256='969c0881050c4a48d19be9ea64bf49fa68c1403b69f9f739bbfd865dda639b2d', 
```
Then `preferred=True` won't be needed anymore.

File: var/spack/repos/builtin/packages/bref3/package.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        install(jar_file, prefix.bin)
+
+        # Set up a helper script to call java on the jar file,
+        # explicitly codes the path for java and the jar file.
+        versions_bref3 = ['25Nov19', '12Jul19']
+        versions_bref  = ['27Jan18']
+        if str(self.version) in versions_bref:
+            script = prefix.bin.bref
+        elif str(self.version) in versions_bref3:
+            script = prefix.bin.bref3
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/bref3/package.py]: Once you update the versions above, you'll be able to use:
```python
if self.version >= Version('2019'):
    script = prefix.bin.bref3
else:
    script = prefix.bin.bref
```

**Comment by xfzhao** [comment]
thank you for your suggestion!

________________________________________

**Comment by adamjstewart** [comment]
Just ping me when the tests pass and I'll merge this.

**Pull request merged**